tools/test-generator/src/ocaml_special_cases.ml

```diff
@@ -147,10 +148,10 @@ let ocaml_edit_parameters ~(slug: string) (parameters: (string * json) list) = m
 | ("change", ps) -> edit_expected ~f:edit_change_expected ps
 | ("connect", ps) -> edit_expected ~f:edit_connect_expected ps
 | ("dominoes", ps) -> edit_dominoes ps
-| ("forth", ps) -> edit_expected ~f:option_of_null ps
+(* | ("forth", ps) -> edit_expected ~f:option_of_null ps *)
 | ("hamming", ps) -> edit_expected ~f:(optional_int ~none:(-1)) ps
 | ("palindrome-products", ps) -> edit_palindrome_products ps
-| ("phone-number", ps) -> edit_expected ~f:option_of_null ps
+(* | ("phone-number", ps) -> edit_expected ~f:option_of_null ps *)
```

Comment:
And `forth` and `phone-number` are exercises that happen to not currently work with test-generator?

Reply:
Yes. Both are affected by issues around option_of_null that require more extensive changes. I'd like to make those this when touching the exercises.